src/Symfony/Bundle/SecurityBundle/Command/UserPasswordEncoderCommand.php

```diff
@@ -94,11 +95,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $encoder = $this->getContainer()->get('security.encoder_factory')->getEncoder($userClass);
         $bcryptWithoutEmptySalt = !$emptySalt && $encoder instanceof BCryptPasswordEncoder;
+        $argon2iWithoutEmptySalt = !$emptySalt && $encoder instanceof Argon2iPasswordEncoder;
```

Comment:
I suggest adding a a marker interface on encoders not needing the salt externally, to avoid having to know them one by one (I have the same need for detecting them in FOSUserBundle 2 to avoid generating useless salts).
what do you think @symfony/deciders ?